sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/DefaultScheduler.java

```diff
@@ -114,6 +103,7 @@
      * such as destroying a failed task.
      */
     protected final int destructiveRecoveryDelayMs;
+    private final Optional<RecoveryPlanManagerFactory> recoveryPlanManagerFactoryOptional;
```

Comment:
It is not.  I was going to break it out, but I thought this PR provided some context, and proof that it works etc.